pyomo/contrib/parmest/parmest.py

```diff
@@ -68,10 +68,14 @@
 from pyomo.common.deprecation import deprecated
 from pyomo.common.deprecation import deprecation_warning
 
+from pyomo.core.base.suffix import Suffix
+
+DEPRECATION_VERSION = "6.7.2.dev0"
+
 parmest_available = numpy_available & pandas_available & scipy_available
 
 inverse_reduced_hessian, inverse_reduced_hessian_available = attempt_import(
-    'pyomo.contrib.interior_point.inverse_reduced_hessian'
+    "pyomo.contrib.interior_point.inverse_reduced_hessian"
```

Comment:
`black` went a little crazy on this file all over the place. Can you please revert those changes? It will make reviewers have to look at a lot more lines that aren't actually any different.